straxen/plugins/nveto_recorder.py

```diff
@@ -71,19 +71,25 @@ def compute(self, raw_records_nv, start, end):
         intervals = coincidence(raw_records_nv,
                                 self.config['coincidence_level_recorder_nv'],
                                 self.config['resolving_time_recorder_nv'])
+        # Always save the first and last resolving_time nanoseconds (e.g. 600 ns)  since we cannot guarantee the gap
+        # size to be larger. (We cannot use an OverlapingWindow plugin either since it requires disjoint objects.)
+        intervals_with_bounds = np.zeros((len(intervals) + 2, 2), dtype=np.int64)
+        intervals_with_bounds[1:-1, :] = intervals
+        intervals_with_bounds[0, :] = start, min(start+self.config['resolving_time_recorder_nv'], intervals[0,0])
```

Comment:
@WenzDaniel , it looks like your idea for https://github.com/XENONnT/straxen/pull/193 is already bearing fruit. I think this is not empty-data proof/

Reply:
:D I am happy that it works. I fixed the issue